logprep/runner.py

```diff
@@ -230,6 +249,15 @@ def reload_configuration(self):
                 + str(error)
             )
 
+    def _schedule_config_refresh_job(self):
+        refresh_interval = self._configuration.get("config_refresh_interval")
```

Comment:
In an error cases you enforce a min interval of 5 seconds. Maybe we should do the same on the initial start up of logprep. Just in case someone should be so creative to use a `config_refresh_interval` of 0 seconds. At the moment this would mean that logprep might try to reload the configuration many times per second.

Reply:
good point -> DONE